### ISSUE TITLE
luci-app-filemanager: Directory incorrectly shown as empty

### ISSUE THREAD

**Issue by KA2107** [openwrt/luci#7454]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### screenshots or captures

![2024-12-05_Luci_File_Manager](https://github.com/user-attachments/assets/2660b5c2-096b-402a-9ab4-bfc8dc4ac4df)

### Actual behaviour

LuCI File Manager incorrectly shows the directory as empty.

### Expected behaviour

LuCI File Manager lists the correct directory contents.

### Steps to reproduce

1) Install luci-app-filemanager
2) Go to LuCI File Manager at http://<ip_addresss>/cgi-bin/luci/admin/system/filemanager

### Additional Information

```shell
NAME="OpenWrt"
VERSION="24.10.0-rc2"
ID="openwrt"
ID_LIKE="lede openwrt"
PRETTY_NAME="OpenWrt 24.10.0-rc2"
VERSION_ID="24.10.0-rc2"
HOME_URL="https://openwrt.org/"
BUG_URL="https://bugs.openwrt.org/"
SUPPORT_URL="https://forum.openwrt.org/"
BUILD_ID="r28161-ea17e958b9"
OPENWRT_BOARD="mediatek/filogic"
OPENWRT_ARCH="aarch64_cortex-a53"
OPENWRT_TAINTS=""
OPENWRT_DEVICE_MANUFACTURER="OpenWrt"
OPENWRT_DEVICE_MANUFACTURER_URL="https://openwrt.org/"
OPENWRT_DEVICE_PRODUCT="Generic"
OPENWRT_DEVICE_REVISION="v0"
OPENWRT_RELEASE="OpenWrt 24.10.0-rc2 r28161-ea17e958b9"
OPENWRT_BUILD_DATE="1733226068"

luci-app-bcp38 - 24.338.58761~e76155d
luci-app-ddns - 24.338.58761~e76155d
luci-app-filemanager - 24.338.58761~e76155d
luci-app-firewall - 24.338.58761~e76155d
luci-app-https-dns-proxy - 2023.12.26-r1
luci-app-irqbalance - 24.338.58761~e76155d
luci-app-openvpn - 24.338.58761~e76155d
luci-app-package-manager - 24.338.58761~e76155d
luci-app-sqm - 24.338.58761~e76155d
luci-app-travelmate - 24.338.58761~e76155d
luci-app-ttyd - 24.338.58761~e76155d
luci-app-uhttpd - 24.338.58761~e76155d
luci-app-wol - 24.338.58761~e76155d
luci-base - 24.338.58761~e76155d
luci-compat - 24.338.58761~e76155d
luci-lib-base - 24.338.58761~e76155d
luci-lib-ip - 24.338.58761~e76155d
luci-lib-jsonc - 24.338.58761~e76155d
luci-lib-nixio - 24.338.58761~e76155d
luci-lib-uqr - 24.338.58761~e76155d
luci-light - 24.338.58761~e76155d
luci-lua-runtime - 24.338.58761~e76155d
luci-mod-admin-full - 24.338.58761~e76155d
luci-mod-dashboard - 24.338.58761~e76155d
luci-mod-network - 24.338.58761~e76155d
luci-mod-status - 24.338.58761~e76155d
luci-mod-system - 24.338.58761~e76155d
luci-proto-batman-adv - 24.338.58761~e76155d
luci-proto-ipv6 - 24.338.58761~e76155d
luci-proto-modemmanager - 24.338.58761~e76155d
luci-proto-ppp - 24.338.58761~e76155d
luci-proto-wireguard - 24.338.58761~e76155d
luci-ssl-openssl - 24.338.58761~e76155d
luci-theme-bootstrap - 24.338.58761~e76155d
```

### What browsers do you see the problem on?

Chrome

### Relevant log output

```shell

```

**Comment by rdmitry0911** [comment]
What theme are you using? Try with a default one. Put here a console log of a new session with a default theme. I've just tried with OpenWrt 24.10.0-rc2 and it works as expected.

**Comment by rdmitry0911** [comment]
I've just made a PR with a fix for this problem. As a workaround you can temporary change your time zone in system settings to Europe

**Comment by systemcrash** [comment]
Closed by #7458

**Comment by WhateverMO** [comment]
![](https://github.com/user-attachments/assets/bcde2698-53b2-4052-8bb9-35347c882c4c)

same issue again why?

![](https://github.com/user-attachments/assets/8605005a-de5d-4553-a437-d691f2f8a85d)

here is the web browser console error output

Failed to parse line: drwxr-xr-x   2 root root  4096 2025-02-03 23:09:37.000000000 +0000 bin
```
zjj@a2d9a9d0ebfb ~/o/o/f/l/a/l/h/l/r/v/system (7b0663a5557118499dc3b3d44550efc1b6fa3feb)> sed -n '300,302p' filemanager.js 
			// Parse the output line from 'ls' command
			var parts = line.match(/^([\-dl])[rwx\-]{2}[rwx\-Ss]{1}[rwx\-]{2}[rwx\-Ss]{1}[rwx\-]{2}[rwx\-Tt]{1}\s+\d+\s+(\S+)\s+(\S+)\s+(\d+)\s+([\d\-]+\s+[\d\:\.]{8,12}\s+[+-]\d{4})\s+(.+)$/);
			if (!parts || parts.length < 7) {
```
it seems there is the bug

and i change the js code ：
```
var parts = line.match(/^([\-dl])[rwx\-]{2}[rwx\-Ss]{1}[rwx\-]{2}[rwx\-Ss]{1}[rwx\-]{2}[rwx\-Tt]{1}\s+\d+\s+(\S+)\s+(\S+)\s+(\d+)\s+([\d\-]+\s+\d{2}:\d{2}:\d{2}(?:\.\d+)?\s+[+-]\d{4})\s+(.+)$/);
```

it work well

![](https://github.com/user-attachments/assets/6ff7f002-60a9-4550-a784-b3106f1da467)

and i will make a PR to solve this bug
